tests/github-workflow.json

```diff
@@ -0,0 +1,1181 @@
+{
```

Comment:
This file comes from https://github.com/SchemaStore/schemastore/blob/15b1babd951184784e608feade64620235f99485/src/schemas/json/github-workflow.json, right?

Delete it and use a git submodule then.